sql/moz-fx-data-shared-prod/mozilla_vpn_derived/waitlist_v1/query.sql

```diff
@@ -0,0 +1,28 @@
+-- EXECUTE IMMEDIATE is necessary to use @date in the query for EXTERNAL_QUERY,
+-- because it requires a string literal
+EXECUTE IMMEDIATE CONCAT(
```

Comment:
it's ugly, but also it works? I'm open to suggestions.

Reply:
```
SELECT FORMAT("DATE '%t'", DATE '2020-01-01')
```

yields `DATE '2020-01-01'`. 

So I would expect we could do:

```

    EXTERNAL_QUERY(
      "moz-fx-guardian-prod-bfc7.us.guardian-sql-prod",
      FORMAT("SELECT * FROM vpn_waitlist WHERE DATE(updated_at) = DATE '%t'", @date))
```

I can't do a full test because I don't have access to the db connection.

Reply:
EXTERNAL_QUERY requires a string literal, it does not allow the query to be generated at runtime:

> Unexpected TVF argument type found, expecting literal string at [17:7]

Reply:
I found out that it accepts a string literal *or query parameter*, so moving the whole external_database_query value to a string query parameter fixed this issue.